### ISSUE TITLE
Enhancing the UI of web pages

### ISSUE THREAD

**Issue by shreyabagade** [Sahil1786/Petari#27]
Hello Sahil1786, the UI of overall pages doesn't look seamless so I would like to enhance and unify the appearance of the pages.

**Comment by Prachiii8287** [comment]
@Sahil1786 yes I also want to enhance UI ,please give me a chance.Please assign this task to me.

**Comment by dhananjaysingh10** [comment]
Hi @Sahil1786, UI of the website is one of the most important things. Having knowledge of the tech stack used in the project and UI/UX designing, I think I can contribute to the project. I, along with other contributors, can make the UI more attractive, engaging, and provide a good user experience. Can you please assign this task to me?

**Comment by IP80808080** [comment]
👋 Hello! I'd be happy to help with this issue. Could you please assign it to me? Additionally, could you please label it for the appropriate level? Thank you! 🚀

**Comment by somrima-09** [comment]
Could you please assign it to me?
I am looking for my 1st contribution
